README.md

```diff
@@ -163,3 +163,23 @@ defined by a composition of a translation and a linear transformation. An
 (and controllable) if you construct it from a composition of a linear map
 and a translation, e.g. `Translation(v) ∘ LinearMap(v)` (or any combination of
 `LinearMap`, `Translation` and `AffineMap`).
+
+#### Perspective transformations
+
+The perspective transformation maps real-space coordinates to those on a virtual
+"screen" of one lesser dimension. For instance, this process is used to render
+3D scenes to 2D images in computer generated graphics and games. It is an ideal
+model of how a pinhole camera operates and is a good approximation of the modern
+photography process.
+
+The `PerspectiveMap()` command creates a singleton `Transformation` to perform
+the projective mapping. It can be applied individually, but is particularly
+powerful when composed with an `AffineMap` containing the position and
+orientation of the camera in your scene. For example, to transfer `points` in 3D
+space to 2D `screen_points` giving their projected locations on a virtual camera
+image, you might use the following code:
+
+```julia
+cam_transform = PerspectiveMap() ∘ inv(AffineMap(cam_rotation, cam_position))
```

Comment:
Good point Andrew. I think it would be very useful to have additional functionality that takes into account the camera parameters such as: focal length, pixel size (x & y) and offset (x & y).

On the other hand I think it would be bad to return to the days of thinking about all these problems as matrix problems. I find that to be both confusing and potentially less efficient than it could be.

I would advocate that we acheive this by composing the "ideal" `PerspectiveMap` with unit-distance screen with a further `AffineMap` to account for the pixel sizing and offsets. This could be achieved by a new function, a simplified version (with square pixels) being:

```julia
function cameramap(focal_length, pixel_size, offset_x, offset_y)
    return AffineMap(UniformScaling(focal_length/pixel_size), SVector(offset_x, offset_y))
         ∘ PerspectiveMap() 
end
```

I suppose the same function could optionally be used to include the camera origin and orientation. I think maybe a keyword-argument function would be pretty great for this.